tests/gdimagecopyresampled/bug00201.c

```diff
@@ -0,0 +1,71 @@
+#include "gd.h"
+#include "gdtest.h"
+
+
+#define SRC_WIDTH 80
+#define SRC_HEIGHT 42
+#define DEST_WIDTH 200
+#define DEST_HEIGHT 200
+
+
+/* Create an image and fill it with true transparency */
+gdImagePtr blank_image(int width, int height)
+{
+    gdImagePtr img;
+    int c;
+
+    img = gdImageCreateTrueColor(width, height);
+    gdImageAlphaBlending(img, 0);
+    c = gdImageColorAllocateAlpha(img, 255, 0, 255, 127);
+    gdImageFilledRectangle(img, 0, 0, width-1, height-1, c);
+    return img;
+}
+
+
+/* Merge layers on top of each other */
+gdImagePtr flatten(gdImagePtr *layers, int layer_count, int width, int height)
+{
+    gdImagePtr img;
+    int i;
+
+    img = blank_image(width,height);
+    gdImageAlphaBlending(img, 1);
+    for (i = 0; i < layer_count; i++) {
+        gdImageCopy(img, layers[i], 0,0, 0,0, width,height);
+    }
+    gdImageSaveAlpha(img, 1);
+    return img;
+}
+
+
+int main()
+{
+    gdImagePtr layers[2], background, logo_source, logo, scaled_logo, img;
+    FILE *fp;
+
+    background = blank_image(DEST_WIDTH,DEST_HEIGHT);
+
+    fp = gdTestFileOpen("gdimagecopyresampled/bug00201_src.png");
+    logo_source = gdImageCreateFromPng(fp);
```

Comment:
please use gdTestFileOpen2 so the directory separator will be handled correctly :)